int8[] angle_offsets
---
bool succes